src/views/collection-view/index.tsx

```diff
@@ -32,18 +34,12 @@ function CollectionView({
         numTutorials={tutorials.length}
       />
       <h2 id={layoutProps.headings[1].slug}>Tutorials</h2>
```

Comment:
Might not be worth tackling in this PR, but we should figure out what to do with the Tutorials heading here.

Reply:
Made a task so we don't forget - https://app.asana.com/0/1202001387788447/1202168098250852